front_end/src/components/icons/die.tsx

```diff
@@ -0,0 +1,30 @@
+import clsx from "clsx";
+
+export function Die({ className, ...props }: React.SVGProps<SVGSVGElement>) {
+  const diePath =
+    "M64 80c-8.8 0-16 7.2-16 16V416c0 8.8 7.2 16 16 16H384c8.8 " +
+    "0 16-7.2 16-16V96c0-8.8-7.2-16-16-16H64zM0 96C0 60.7 28.7 32 " +
+    "64 32H384c35.3 0 64 28.7 64 64V416c0 35.3-28.7 64-64 64H64c-35.3 " +
+    "0-64-28.7-64-64V96zm128 32a32 32 0 1 1 0 64 32 32 0 1 1 0-64zm0 " +
+    "192a32 32 0 1 1 0 64 32 32 0 1 1 0-64zm64-64a32 32 0 1 1 64 0 32 32 " +
+    "0 1 1 -64 0zM320 128a32 32 0 1 1 0 64 32 32 0 1 1 0-64zM288 352a32 32 " +
+    "0 1 1 64 0 32 32 0 1 1 -64 0z";
```

Comment:
nit: define this outside of the component, so `diePath` variable is not re-initialized on every re-render:

```
const `DIE_PATH = ""

export function Die ...
```